pkg/kubelet/types/labels.go

```diff
@@ -21,6 +21,7 @@ const (
 	KubernetesPodNamespaceLabel  = "io.kubernetes.pod.namespace"
 	KubernetesPodUIDLabel        = "io.kubernetes.pod.uid"
 	KubernetesContainerNameLabel = "io.kubernetes.container.name"
+	KubernetesContainerdPauseNameLabel = "io.cri-containerd.kind"
```

Comment:
Why `PauseName` instead of `Name`?

Reply:
Cause only pause container doesn't have container name label for containerd. Other containers in containerd just use KubernetesContainerNameLabel.

Reply:
Still, this label looks generic enough to have a generic name such as `ContainerdKindLabel` or `CriContainerdKindLabel` don't you think?

Reply:
Yes, i will correct it.

Reply:
Done~